week-5/ghostdb/Scripts/3_script_data_table.sql

```diff
@@ -0,0 +1,24 @@
+--INSERT DATA IN TABLE PRODUCTS
+insert into products(product_name, days, price)
+select distinct product_name, days, price from temporal;
+
+--INSERT DATA IN TABLE CUSTOMERS
+insert into customers(customer_name, customer_city, customer_state, customer_country)
+select distinct customer_name, customer_city,customer_state,customer_country
+from   temporal;
+
+--INSERT DATA IN TABLE SALES
+insert into sales(product_id, customer_id,transaction_date,payment_type,sold_by,contact,pos)
+select 	p.product_id,
+		c.customer_id,
+		t.transaction_date,
+		t.payment_type,
+		t.sold_by,
+		t.contact,
+		t.pos
+from 	temporal t
```

Comment:
I would like to know about 'temporal'

Reply:
Temporal reference the intermediate table in our project.

Reply:
We didn't use a a ruby files to generate each script of inserting data to each table, we used a intermediary table have the same structure of file and generating an unique script of inserting data from ruby. Then from that intermediary table we create the rows of each main table by using insert massive with select to that table.

Reply:
Thanks, now I get it.